src/hotspot/share/gc/shenandoah/shenandoahHeap.hpp

```diff
@@ -317,7 +317,10 @@ class ShenandoahHeap : public CollectedHeap {
     WEAK_ROOTS_BITPOS  = 4,
 
     // Old regions are under marking, still need SATB barriers.
-    OLD_MARKING_BITPOS = 5
+    OLD_MARKING_BITPOS = 5,
+
+    // The evacuation reserves for old-gen and young-gen are available
+    VALID_EVACUATION_RESERVE_QUANTITIES_BITPOS = 6
```

Comment:
I think we only do these when we want to access the flags from generated code. If we only do the accesses from the GC code, this could be just a plan `bool` field.